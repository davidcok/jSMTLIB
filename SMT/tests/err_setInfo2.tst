; malformed set-info commands
(set-info )
(set-info x y)
(set-info :x )
(set-info :x :y )
(set-info :x y y)
