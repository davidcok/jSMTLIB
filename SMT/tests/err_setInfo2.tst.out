(set-info )
          ^
(error "Expected a keyword (beginning with a colon) here, instead of a )")
(set-info x y)
          ^
(error "Expected a keyword (beginning with a colon) here, instead of a symbol")
(set-info :x )
             ^
(error "Expected an attribute value here, instead of a )")
(set-info :x :y )
             ^^
(error "Expected an attribute value here, instead of a keyword")
(set-info :x y y)
               ^
(error "Too many arguments or extraneous material after the command or missing right parenthesis")
